rocAL/include/pipeline/commons.h

```diff
@@ -68,11 +70,13 @@ enum class RocalTensorDataType {
     INT32,
     INT16
 };
+REGISTER_ENUM(RocalTensorDataType)
 
 enum class RocalAffinity {
```

Comment:
Non-blocking: Since this is an "enum class", consider explicitly specifying the underlying type (e.g., "enum class <Name> : int") to keep ABI stable and avoid surprises where different compilers choose different underlying types.